backend/src/openarchiefbeheer/destruction/tests/test_models.py

```diff
@@ -285,3 +285,44 @@ def test_assign_next_long_process(self):
 
         self.assertEqual(destruction_list.status, ListStatus.internally_reviewed)
         self.assertEqual(destruction_list.assignee, destruction_list.author)
+
+    def test_assign_next_archivist(self):
+        archivist = UserFactory.create(
+            role__can_review_final_list=True,
+        )
+        reviewer = UserFactory.create(
+            role__can_review_destruction=True,
+        )
+        destruction_list = DestructionListFactory.create(
+            assignee=archivist, status=ListStatus.ready_for_archivist
+        )
+        zaken = ZaakFactory.create_batch(
+            2, zaaktype="http://catalogi-api.nl/zaaktype/1"
+        )
+        DestructionListItemFactory.create(
+            destruction_list=destruction_list, zaak=zaken[0].url
+        )
+        DestructionListItemFactory.create(
+            destruction_list=destruction_list, zaak=zaken[1].url
+        )
```

Comment:
I think that this can be simplified to 
```suggestion
        DestructionListItemFactory.create_batch(2, destruction_list=destruction_list)
```
Since the specifics of the zaken don't matter in this test case.